images/kindnetd/go.mod

```diff
@@ -1,49 +1,64 @@
 module sigs.k8s.io/kind/images/kindnetd
 
-go 1.21
+go 1.22.0
 
 require (
 	github.com/coreos/go-iptables v0.7.0
 	github.com/vishvananda/netlink v1.1.0
-	k8s.io/api v0.29.0
-	k8s.io/apimachinery v0.29.0
-	k8s.io/client-go v0.29.0
-	k8s.io/klog/v2 v2.110.1
+	golang.org/x/sys v0.22.0
+	k8s.io/api v0.30.2
+	k8s.io/apimachinery v0.30.2
+	k8s.io/client-go v0.30.2
+	k8s.io/klog/v2 v2.130.1
+	sigs.k8s.io/kube-network-policies v0.5.0
 )
 
 require (
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.3.0 // indirect
```

Comment:
I assume that you've done the diligence on these and their licenses etc?

looks like most of these are not new but a few are

Reply:
github.com/beorn7/perks, https://github.com/cespare/xxhash github.com/mdlayher/netlink and github.com/florianl/go-nfqueue are MIT